content/docs/fairing/azure/_index.md

```diff
@@ -0,0 +1,121 @@
++++
+title = "Fairing on Azure"
+description = "Documentation for Kubeflow Fairing on Azure Kubernetes Service (AKS)"
+weight = 45
++++
+
+This page documents how to run the [Fairing prediction sample notebook](https://github.com/kubeflow/fairing/blob/master/examples/prediction/xgboost-high-level-apis.ipynb) on Azure in a notebook hosted in Kubeflow.
+
+While the documentation shows how to run the sample on [Azure Kubernetes Service (AKS)](https://azure.microsoft.com/en-in/services/kubernetes-service/), the steps should work on any standard Kubernetes cluster.
+
+## Prerequisites
+
+Before you configure and use the Kubeflow Fairing Python SDK, you must have a Kubeflow environment installed. The sample notebook does in-cluster training and thus it is not recommended to use the smallest size virtual machines. The sample has been tested to work in a cluster with 2 nodes and Standard_D4_v3 size.
+
+* If you do not have a Kubeflow installation on Azure, follow the [installation guide][/docs/azure/deploy/install-kubeflow]
+* You must have the `kubectl` CLI installed and configured to use the Kubernetes cluster where Kubeflow is installed (in above installation guide, the command `az aks get-credentials` takes care of that)
+
+## Create Azure Container Registry and Storage
+
+In this notebook sample, Azure Container Registry (ACR) is used to host docker images for deployment and Azure Storage (Storage) is used as build context for in-cluster building.
+
+You can re-use existing ACR and Storage or create new ones. For more information how to create, please see the documentation for [ACR](https://docs.microsoft.com/en-us/azure/container-registry/) and [Storage](https://docs.microsoft.com/en-us/azure/storage/).
```

Comment:
Nit: Consider changing this to be an ordered list, since this is a sequential set of instructions.